src/mlpack/tests/loss_functions_test.cpp

```diff
@@ -320,6 +321,48 @@ TEST_CASE("SimpleBinaryCrossEntropyLossTest", "[LossFunctionsTest]")
   REQUIRE(output.n_cols == input2.n_cols);
 }
 
+/*
+ * Simple test for the binary-cross-entropy with Logits loss function.
+ */
+TEST_CASE("SimpleBinaryCrossEntropyWithLogitsLossTest", "[LossFunctionsTest]")
+{
+  arma::mat input1, input2, input3, output, target1, target2, target3;
+  BCELossWithLogits<> module1(1e-6, false);
+  BCELossWithLogits<> module2(1e-6, true);
+  // Test the Forward function on a user generator input and compare it against
+  // the manually calculated result.
+  input1 = arma::mat("0.5 0.5 0.5 0.5 0.5 0.5 0.5 0.5");
+  target1 = arma::zeros(1, 8);
```

Comment:
```suggestion
  target1 = arma::ones(1, 8);
```

I checked the test cases using `ones` .